packages/ui/components/EligibleSection/EligibleSection.tsx

```diff
@@ -1,10 +1,24 @@
 import { Button } from 'components/Button';
 import React from 'react';
+import axios from 'axios'
+import { useRouter } from 'next/router';
 
 function EligibleSection({
   setShowEligibleSection,
   isShowEligibleSection,
 }: any) {
+  const router = useRouter()
+  async function createProgram() {
+    const createProgramResponse = await axios.post('/programs', {
```

Comment:
Any reason why we are using axios? instead of default fetch.

Reply:
I find I always end up wrapping fetch and turning it into a mini-library within the code.